docs/concepts/build-backend.md

```diff
@@ -54,7 +54,25 @@ module-name = "PIL"
 module-root = ""
 ```
 
-The build backend supports building stubs packages with a `-stubs` package or module name.
+For a namespace packages, the path can be dotted. The example below expects to find a
+`src/cloud/db/schema/__init__.py`:
+
+```toml
+[tool.uv.build-backend]
+module-name = "cloud.db.schema"
+```
+
+Complex namespaces with more than one root module can be built by setting the `namespace` option.
+Note that this option disables most coherence check, but it is still required to have a valid
```

Comment:
Is it feasible to give an example of something that will be accepted here that normally wouldn't be? Reading this with my beginner eyes on (as if I didn't just read this PR), I become immediately curious what kinds of checks have been disabled on me. :-) Should I be concerned? What if I want those checks enabled?

Reply:
I've put a comparison into the reference docs.